docs: v1.5.3 changelog

alexmt opened the pull request.

jannfis commented on CHANGELOG.md:
Missing issue link

alexmt replied:
fixed

jannfis commented on CHANGELOG.md:
Missing issue link

alexmt replied:
fixed

jannfis commented on CHANGELOG.md:
Missing issue link

alexmt replied:
fixed

jannfis commented on CHANGELOG.md:
Maybe the upgrade of bundled Kustomize should also be noted in the `Multiple Kustomize Versions` section above, in a side sentence.

alexmt commented:
Addressed your comments @jannfis . I've also noticed that customize documentation was incomplete and cli flag name had a typo. Fixed in as well. Please take another look
